SECURITY.md

```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+Mozilla has a [well-defined process for handling security vulnerabilities](https://www.mozilla.org/en-US/about/governance/policies/security-group/bugs/) based around responsible disclosure.
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| > 31.x  | :white_check_mark: |
+| < 15.0  | :x:                |
```

Comment:
Just to see if I understand this, we support greater than major version 31 and don't support less than major version 15. What about major versions between 15 and 31?

Reply:
That's a leftover from kinto 🤦